backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP cannot be used as a disaster recovery solution for etcd or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
```

Comment:
". . . information on" is used, but ". . . information about" is more commonly used in our docs:

I have a few suggestions here for how better to word this for clarity, because I don't think the link necessarily provides information about "etcd", but rather "backing up etcd".

1:
```suggestion
OADP cannot be used as a disaster recovery solution for etcd or OpenShift Operators. See the documentation for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd] for more information. 
```
2:
```suggestion
For more information about backing up etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
```

That said, I'm not really crazy about including the link to this information right here. I think that it could go on the bottom of the page, after the content, in an additional resources section. See comment below for example. 

Kind of up to you how you want to proceed.